packages/db/fauna/schema.graphql

```diff
@@ -514,11 +544,17 @@ input CreateUploadInput {
   cid: String!
   type: UploadType!
   name: String
+  backupData: [createBackupInput]!
```

Comment:
Minor: `backupData` is not actually the data that is being backed up, just information about where it can be found.

Maybe `backupLocations` or `backupUrls` if you switch to using a URL instead.